frontend/packages/text-editor/src/utils.ts

```diff
@@ -89,3 +92,27 @@ export const validateTextId = (textIdToValidate: string): string => {
 
   return '';
 };
+
+export const autosize = (element) => {
```

Comment:
Could we simplify this code by creating a hook instead, `useAutosizeTextarea`? Please take a look at this Sandbox, https://codesandbox.io/s/autosize-textarea-forked-xlqm3k?file=/src/useAutosizeTextArea.ts. :) 

What do you think? 🤔 😄